keel-core-test/src/main/kotlin/com/netflix/spinnaker/keel/persistence/ResourceRepositoryTests.kt

```diff
@@ -302,17 +361,23 @@ abstract class ResourceRepositoryTests<T : ResourceRepository> : JUnit5Minutests
             .isFailure()
             .isA<IllegalArgumentException>()
         }
+
+        test("event history includes relevant application events") {
+          expectThat(subject.eventHistory(resource.id)).and {
```

Comment:
I think you should be able to omit `.and` there. There's an override of `expectThat` taking a lambda as a 2nd parameter.

Reply:
Done.